pom.xml

```diff
@@ -46,7 +46,10 @@
 	<properties>
 		<jna.version>5.13.0</jna.version>
 		<junit.version>4.13.1</junit.version>
+		<test.plugin.version>3.2.3</test.plugin.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<integration.test.model>mistral-7b-instruct-v0.2.Q5_K_S.gguf</integration.test.model>
```

Comment:
I decided to rework this to operate with mistral 7b since it's smaller and apache licensed.